src/components/memorygame/MemoryGameCards/index.jsx

```diff
@@ -0,0 +1,163 @@
+import React, { useState, useEffect } from "react";
+import "./index.css";
+import ReactCardFlip from "react-card-flip";
+import { Col, Card, Modal, Row } from "antd";
+import { useTranslation } from "react-i18next";
+import questionMark from "../../../images/memoryGameFrontImage.png";
+import memoryGameSuccessSymbol from "../../../images/memoryGameSuccessSymbol.svg";
+import cardInitialStateGenerator from "../cardInitialStateGenerator";
+
+const NUM_REQUIRED_MATCHES = 2;
+// game example: https://www.brightfocus.org/alzheimers/memory-games/match-pictures
+// instead of squares this game uses cards.
+// Card flips on click to show what is behind.
+// matchedCards is an array states that holds successfully matched card states
+// setCompleted is true when all possible matches made
```

Comment:
set completed is a function, so the sentence that it is true (a boolean) doesn't make sense

Reply:
You are right. Also, it should be `setIsCompleted`.

// `setIsCompleted` is a function that is being used to set `isCompleted` as boolean state when all possible matches made.

I think this is better

Reply:
PTAL